### ISSUE TITLE
output too small error on new preprod network

### ISSUE THREAD

**Issue by amirmrad** [Plutonomicon/cardano-transaction-lib#977]
Running SendToken example on [preprod](https://github.com/Plutonomicon/cardano-transaction-lib/pull/944) causes an output too small on submission to ogmios.
```
Error from Ogmios: [{"outputTooSmall":[{"address":"addr_test1qqrjnzzvd7wygv3qjjhm7l574zuu6vrgl0k40vv97nkzx88uu4rk8gteywrllxxlg5ylxdxmhppxrx8rmn6y3prk9h5sdwyask","datum":null,"datumHash":null,"value":{"assets":{"fda1b6b487bee2e7f64ecf24d24b1224342484c0195ee1b7b943db50.546865546f6b656e":1},"coins":1155080}}]}]
```

**Comment by amirmrad** [comment]
I suspect there is an issue with `utxoMinAdaValue` calculation

**Comment by ngua** [comment]
I'm assuming this was closed by #981?
